output/transform/postgres_statements.go

```diff
@@ -67,11 +67,27 @@ func transformQueryStatistic(stats state.DiffedPostgresStatementStats, idx int32
 	}
 }
 
-func transformPostgresStatements(s snapshot.FullSnapshot, newState state.PersistedState, diffState state.DiffState, transientState state.TransientState, roleOidToIdx OidToIdx, databaseOidToIdx OidToIdx) snapshot.FullSnapshot {
+type postgresStatementKey struct {
+	databaseOid state.Oid
+	userOid     state.Oid
+	queryID     int64
+}
+type StatementKeyToIdx map[postgresStatementKey]int32
+
+func transformPostgresStatements(s snapshot.FullSnapshot, newState state.PersistedState, diffState state.DiffState, transientState state.TransientState, roleOidToIdx OidToIdx, databaseOidToIdx OidToIdx) (snapshot.FullSnapshot, StatementKeyToIdx) {
 	// Statement stats from this snapshot
 	groupedStatements := groupStatements(transientState.Statements, diffState.StatementStats)
+	queryIdxMap := make(StatementKeyToIdx)
```

Comment:
Could you rename this variable to `statementKeyToIdx`? That way it's consistent across all uses in this PR. Or the other direction is fine to, renaming all usage to `queryIdxMap`. I'd just like it to be consistent to make it easier to navigate the code.